README.md

```diff
@@ -102,6 +102,7 @@
 - [HitBTC](https://hitbtc.com/NEO-to-ETH) - A European exchange that allows trading for BTC, ETH and USDT.
 - [JUBI](https://www.jubi.com/coin/ans) - A Chinese exchange that allows trading for Chinese Yuan Renminbi (CNY). *Chinese interface*
 - [Poloniex](https://www.19800.com/trade.html?marketid=cny_gas) - A United States-based exchange that allows trading for BTC and ETH (only GAS).
+- KuCoin [GAS](https://www.kucoin.com/#/trade/GAS-BTC) [NEO](https://www.kucoin.com/#/trade/NEO-BTC) - Korean Exchange doing GAS-BTC and NEO-BTC.
```

Comment:
1. Please add a single link, following the format of all other links.
1. Please place the link in A-Z alphabetical order.